src/components/HamburgerMenu/HamburgerMenu.tsx

```diff
@@ -0,0 +1,25 @@
+import { useHamburgerMenu } from 'hooks/useHamburgerMenu';
+import { HamburgerButton, StyledIcon } from './styled';
+import MenuContainer from './MenuContainer';
+
+export default function HamburgerMenu() {
+  const { isOpen, openMenu, closeMenu } = useHamburgerMenu();
+
+  const handleHamburgerClick = () => (isOpen ? closeMenu() : openMenu());
```

Comment:
You can simplify this ternary operations

`const handleHamburgerClick = isOpen ? closeMenu : openMenu;`